R/summary_sum.R

```diff
@@ -0,0 +1,35 @@
+#'  Sum
```

Comment:
@chariga98 can you put a more informative title - such as "Calculate sum" or something?

Reply:
ok I will